icepond/ted/uturn.hoon

```diff
@@ -0,0 +1,18 @@
+/-  spider, uturn, icepond
+/+  icepond, strandio
+=,  strand=strand:spider
+^-  thread:spider
+|=  arg=vase
+=/  m  (strand ,vase)
+^-  form:m
+;<  now=@da  bind:m  get-time:strandio
+;<  eny=@uv  bind:m  get-entropy:strandio
+=/  id  (crip "{(scow %da now)}-{(scow %uv (sham 2 eny))}")
```

Comment:
It's probably good practice to include a thread name or other readable identifier in the thread id, for easier debugging legibility.

```suggestion
=/  id  (crip "uturn-{(scow %da now)}-{(scow %uv (sham 2 eny))}")
```